modules/prediction/container/obstacles/obstacles_container.cc

```diff
@@ -169,13 +174,16 @@ void ObstaclesContainer::InsertPerceptionObstacle(
              << ", and prediction_id: " << prediction_id;
     }
   }
+  curr_frame_id_perception_obstacle_map_[prediction_id] = perception_obstacle;
   if (prediction_id < -1) {
     AERROR << "Invalid ID [" << prediction_id << "]";
     return;
   }
   if (!IsPredictable(perception_obstacle)) {
     ADEBUG << "Perception obstacle [" << prediction_id
            << "] is not predictable.";
+    curr_frame_non_predictable_obstacle_ids_.push_back(
+        perception_obstacle.id());
```

Comment:
Here, the "id" is prediction_obstacle_id or "perception_obstacle_id"? It looks like the two are mixed.

Reply:
Thanks, I will fix it.